tools/python/boutiques/bosh.py

```diff
@@ -81,9 +82,11 @@ def execute(*params):
                                   "changeUser"         : results.user})
         executor.readInput(inp)
         # Execute it
-        exit_code, _, err_msg = executor.execute(results.volumes)
-        if exit_code:
-            raise SystemExit(err_msg)
+        exit_code, stdout, stderr = executor.execute(results.volumes)
+        print("Exit code: {}".format(exit_code))
+        print("Standard output: \"{}\"".format(stdout))
+        print("Standard error: \"{}\"".format(stderr))
+        return (exit_code, stdout, stderr)
```

Comment:
I believe it's better to return all 3. 
As an example it could be useful to capture the pipeline's output.

However, in the context of bosh test, only the exit code is needed.

Reply:
Great, I agree :)